### DISCUSSION TITLE
403 on admin/systems/files preventing Picture and Cropper from working

### DISCUSSION THREAD

**Discussion by Vlad-Galeev** [orchidsoftware/platform#2568] [Q&A]
I am working on a project, and I have ran into an issue with image uploads where whenever I crop or upload an image, the POST request that Orchid automatically sends to `http://127.0.0.1:8000/admin/systems/files` returns a 403 error. Where in permissions would I fix that?

**Comment by BENRAHAL-Aicha** [comment]
same here

**Reply by BENRAHAL-Aicha** [reply]
<img width="332" height="138" alt="image" src="https://github.com/user-attachments/assets/8a4500f8-c111-4cee-813a-c744b4a58ae3" />
you need to go to users -> update -> permissions -> attachement 
<img width="1469" height="325" alt="image" src="https://github.com/user-attachments/assets/4d5869bb-324c-4ff2-a20f-0412ce57b7a3" />
